charts/osm/crds/config_meshconfig.yaml

```diff
@@ -208,10 +208,10 @@ spec:
                   properties:
                     enableWASMStats:
                       type: boolean
-                      default: true
+                      default: false
                     enableEgressPolicy:
                       type: boolean
-                      default: true
+                      default: false
                     enableMulticlusterMode:
                       type: boolean
                       default: false
```

Comment:
These defaults do not convey the correct intention. Based on our discussions, it's easier to always derive defaults from `values.yaml` and configure them via presets.